packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -128,6 +142,7 @@ public void onProgressChanged(WebView view, int progress) {
       String userAgent = (String) params.get("userAgent");
       updateUserAgent(userAgent);
     }
+    if (commonIniter != null) commonIniter.initWebView(webView);
```

Comment:
At present, websettings lacks many functions, and its behavior is quite different from that of IOS. It is recommended to agree to the change of PR and wait until webview_flutter had all these settings are satisfied, the  `initWebView` method can be deprecated.